exchanges/lbank/lbank_wrapper.go

```diff
@@ -756,22 +732,16 @@ func (l *Lbank) GetFeeByType(ctx context.Context, feeBuilder *exchange.FeeBuilde
 		return feeBuilder.Amount * feeBuilder.PurchasePrice * 0.002, nil
 	}
 	if feeBuilder.FeeType == exchange.CryptocurrencyWithdrawalFee {
-		withdrawalFee, err := l.GetWithdrawConfig(ctx,
-			feeBuilder.Pair.Base.Lower().String())
+		withdrawalFee, err := l.GetWithdrawConfig(ctx, feeBuilder.Pair.Base)
 		if err != nil {
 			return resp, err
 		}
 		for i := range withdrawalFee {
-			if !strings.EqualFold(withdrawalFee[i].AssetCode, feeBuilder.Pair.Base.String()) {
+			if !withdrawalFee[i].AssetCode.Equal(feeBuilder.Pair.Base) {
```

Comment:
Couldn't we use just == operation here?

Reply:
If both are Generated from case similar strings they will have same value.
![Uploading Screenshot 2025-05-26 at 4.30.01 in the afternoon.png…]()

Reply:
For case sensitive things, it's a good habit to default to this and enforce even though it's possible to use `==` like you said. This being the less error prone option:

```
func TestCode(t *testing.T) {
	a := currency.NewCode("btc")
	b := currency.BTC
	t.Log(a.Equal(b))
	t.Log(a == b)
}

```

Reply:
Cool. I'll follow this